### DISCUSSION TITLE
Github Student Developer Pack discontinued. Unable to reapply with the same student ID.

### DISCUSSION THREAD

**Discussion by kxenak** [community/community#107451] [GitHub Education]
### Hi everyone,

My github student developer pack has been discontinued. I am trying to reappy with my student ID card but it shows that I am not allowed to reuse a document. I don't know what to do. Can I reapply with another document. I don't know what other document I can use or why my ID card is not being accepted as it is still valid.

**Comment by Daniel3487** [comment]
The discontinuation of the Github Student Developer Pack can pose challenges for students seeking access to valuable tools and resources. If you find yourself unable to reapply with the same student ID, it's advisable to reach out directly to GitHub's support team for assistance. In the meantime, exploring alternative resources or platforms that support   [Mexican Weight Loss Pills](https://mexicanweightlosspills.com/)        educational endeavors, similar to how one might look for alternatives or solutions like 'Ozempic South Africa' for specific needs, could prove beneficial. Diversifying your toolkit and seeking advice from educational forums or communities can also offer viable paths forward.
